workshops/collaborative_sketch/README.md

```diff
@@ -48,23 +54,49 @@ And we'll add script tags for Firebase, p5.js, jQuery, and our own `main.js` fil
 
 ### Setting up the Firebase App
 
-Much like we did in the last workshop, we'll need to create an app within Firebase. In this project, we'll be using Firebase to store the points of what we're drawing on our canvas.
+Firebase is a real-time database. This means that all the information we send to Firebase gets updated across everything connected to that database.
+
+In this project, we'll be using Firebase to store the points of what we're drawing on our canvas. We'll need to create an app within Firebase.
+
+Now we'll set up the Firebase side of our project. Go to [Firebase](https://firebase.google.com/) and sign up for an account.
```

Comment:
Transition is a bit weird - instead of saying "Now we'll set up the Firebase...etc" could we change that to "Let's set up the Firebase...etc"